cmd/stack.go

```diff
@@ -246,6 +248,39 @@ func setupStackCommand() *cobraext.Command {
 	}
 	dumpCommand.Flags().StringP(cobraext.StackDumpOutputFlagName, "", "elastic-stack-dump", cobraext.StackDumpOutputFlagDescription)
 
+	statusCommand := &cobra.Command{
+		Use:   "status",
+		Short: "Show status of the stack services",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			profileName, err := cmd.Flags().GetString(cobraext.ProfileFlagName)
+			if err != nil {
+				return cobraext.FlagParsingError(err, cobraext.ProfileFlagName)
+			}
+			profile, err := profile.LoadProfile(profileName)
+			if err != nil {
+				return errors.Wrap(err, "error loading profile")
+			}
+
+			stackVersion, err := cmd.Flags().GetString(cobraext.StackVersionFlagName)
+			if err != nil {
+				return cobraext.FlagParsingError(err, cobraext.StackVersionFlagName)
+			}
+
```

Comment:
`--version` parameter is needed in order to be able to load the needed environment variables.

```
 $ elastic-package stack status -v
2022/08/11 12:35:13 DEBUG Enable verbose logging
2022/08/11 12:35:13 DEBUG Distribution built without a version tag, can't determine release chronology. Please consider using official releases at https://github.com/elastic/elastic-package/releases
2022/08/11 12:35:13 DEBUG running command: /home/mariorodriguez/bin/docker-compose version --short
2022/08/11 12:35:14 DEBUG Determined Docker Compose version: 1.29.2, the tool will use Compose V1
2022/08/11 12:35:14 DEBUG running command: /home/mariorodriguez/bin/docker-compose -f /home/mariorodriguez/.elastic-package/profiles/default/stack/snapshot.yml -p elastic-package-stack ps -q
ERROR: Couldn't find env file: /home/mariorodriguez/.elastic-package/profiles/default/stack/elastic-agent..env
2022/08/11 12:35:14 ERROR is Elastic stack created/running?
Error: failed getting stack status: running command failed: exit status 1
```

This fails when running docker-compose v1.
In case of docker-compose v2 it does not fail, not sure yet what it is the difference